packages/prisma/extensions/usage-tracking.ts

```diff
@@ -0,0 +1,32 @@
+import { Prisma } from "@prisma/client";
+
+import LicenseKeyService, { UsageEvent } from "@calcom/ee/common/server/LicenseKeyService";
+
+export function usageTrackingExtention() {
+  return Prisma.defineExtension({
+    query: {
+      booking: {
+        async create({ args, query }) {
```

Comment:
Do we cover reschedules and cancellations as discussed before?

Reply:
Not just yet - i will handle this after this is live. 

Limits are so high in self hosted that we thought it wouldnt be an issue for the time being